android/quest/src/main/assets/configs/app/sync_config.json

```diff
@@ -17,13 +17,11 @@
     {
       "resource": {
         "resourceType": "SearchParameter",
-        "name": "organization",
-        "code": "managing-entity.patient.organization",
+        "name": "_id",
+        "code": "_id",
         "base": [
           "Group"
-        ],
-        "type": "token",
-        "expression": "#organization"
+        ]
```

Comment:
Is this intentional? and what's the reason for this change?

Reply:
Yea this was done intentionally (as a workaround) so as to sync all the Group resources including those used in stock.